openapi.yml

```diff
@@ -213,6 +218,8 @@ components:
         wte:
           type: number
           format: percent
+          minimum: 0
+          maximum: 1
```

Comment:
I've been using 0-100 range on FE for percentage (so I don't have to manipulate decimal value to display as percent). Is it better for BE to work with 0-1 range?

Reply:
It is a more standard approach, though if we're going to end up with 60.0000000007% I'm open to the idea of transforming it on the BE 😆

Reply:
(It's also consistent with the trainee profile's WTE field)

Reply:
And look at the fun we can have with that! 😅
I see the `placement`  `wholeTimeEquivalent` is e.g. `"1.0"` 😎